file_transfer/send_file.py

```diff
@@ -36,3 +36,7 @@
 
     sock.shutdown(1)
     sock.close()
+
+
+if __name__ == "__main__":  # pragma: no cover
+    run_script()
```

Comment:
These changes seem like needless clutter.  Do they improve anything?

Reply:
Nevermind...  Now I understand.  I think we can lose the pragma.  We are not aiming for 100% test coverage.